harvester_e2e_tests/apis/test_images.py

```diff
@@ -136,6 +137,56 @@ def test_create_with_reuse_display_name(self, api_client, unique_name, fake_imag
         )
 
         _ = api_client.images.delete(unique_name)
+        endtime = datetime.now() + timedelta(seconds=wait_timeout)
+
+        while endtime > datetime.now():
+            code, data = api_client.images.get(unique_name)
+            if code == 404:
+                break
+            sleep(5)
+        else:
+            raise AssertionError(
+                f"Failed to delete image {unique_name} with {wait_timeout} timed out\n"
+                f"Still got {code} with {data}"
+            )
+
+    @pytest.mark.dependency(name="create_image_url")
+    def test_create_with_url(self, request, api_client, unique_name):
```

Comment:
as the test case requires external resource, we should add it as integration test case.
to make the test steps more clear, we should prevent to use `request` besides other fixtures, we can make a fixture `opensuse_image` for it.

Reply:
removed external resource and changed to use opensuse_image fixture

Reply:
Were you referring to the part? I removed that, but it just occurred to me that you might be referring to the external image (in this case opensuse)?